lib/ic/test/c_client_erl_server_SUITE_data/c_client.c

```diff
@@ -276,7 +276,7 @@ int main(int argc, char **argv)
     char *peer_node = NULL;
     char *peer_process_name = NULL;
     char *cookie = NULL;
-    char host[HOSTNAMESZ + 1];
+    char host[HOSTNAMESZ+1];
```

Comment:
Please don't do changes that only modify the code style. Before this is merged I'll ask you to squash all commits, please make sure that only non-whitespace changes are left.